server/storage/mvcc/watchable_store_test.go

```diff
@@ -164,6 +164,77 @@ func TestSyncWatchers(t *testing.T) {
 	}
 }
 
+func TestRangeEvents(t *testing.T) {
+	b, _ := betesting.NewDefaultTmpBackend(t)
+	lg := zaptest.NewLogger(t)
+	s := NewStore(lg, b, &lease.FakeLessor{}, StoreConfig{})
+
+	defer cleanup(s, b)
+
+	foo1 := []byte("foo1")
+	foo2 := []byte("foo2")
+	foo3 := []byte("foo3")
+	value := []byte("bar")
+	s.Put(foo1, value, lease.NoLease)
+	s.Put(foo2, value, lease.NoLease)
+	s.Put(foo3, value, lease.NoLease)
+	s.DeleteRange(foo1, foo3) // Deletes "foo1" and "foo2" generating 2 events
```

Comment:
Can you add a comment something like below 

https://github.com/etcd-io/etcd/blob/afb5c1d85b2f0ec24852c4d409bca8401dbdea6b/server/storage/mvcc/index_test.go#L240-L256

or just a list of revisions 
```
rev: 1, key: foo1, PUT
rev: 2, key: foo2, PUT
rev: 3, key: foo3, PUT
rev: 4, key: foo1, DELETE
rev: 5, key: foo2, DELETE
```
 so that others can understand the following case more easily
?

Reply:
PTAL https://github.com/etcd-io/etcd/pull/18981

Reply:
Done